srml/system/src/lib.rs

```diff
@@ -201,20 +257,27 @@ fn hash69<T: AsMut<[u8]> + Default>() -> T {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as System {
-
+		/// Extrinsics nonce for accounts
 		pub AccountNonce get(account_nonce): map T::AccountId => T::Index;
-
+		/// Total extrinsics count for the current block
 		ExtrinsicCount: Option<u32>;
+		/// Total length in bytes for all extrinsics put together, for the current block
 		AllExtrinsicsLen: Option<u32>;
+		/// Maps block hashes to block numbers
```

Comment:
does the order here matter? `map T::BlockNumber => T::Hash` maps block numbers to block hashes no?

Reply:
not really, its a 1:1 mapping and the storage entry is named `BlockHash`